news/2025/2025-09-30-moscow-osu-event-2025-recap.md

```diff
@@ -0,0 +1,131 @@
+The tournament had a single-elimination bracket and began from Round of 16.
+
+### Match highlights:
+
+### ::{ flag=RU }:: [BJIADOC](https://osu.ppy.sh/users/17031854) vs. ::{ flag=RU }:: [Lexu2S](https://osu.ppy.sh/users/8251785) 
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/118776739)**\
+**[You can view the match VOD here!](https://youtu.be/dR9aCY2Bq4w?si=VWFEB4ohQAst7xqj&t=2870)**
+
+
+The first match of the tournament, unlike most of viewers expectations, became the first (and only) match that went up to Tiebreaker!
+::{ flag=RU }:: [Lexu2S](https://osu.ppy.sh/users/8251785) was very good on his speed picks, as well as the flow stream one. ::{ flag=RU }:: [BJIADOC](https://osu.ppy.sh/users/17031854), on other side, showed himself well on the snap alt and hidden mod picks, with HD3 being the only close map (14,627 points difference!) up until tiebreaker. The tiebreaker itself was even closer though! With a total difference of 13,725 points, ::{ flag=RU }:: [Lexu2S](https://osu.ppy.sh/users/8251785) have managed to win this match and advance to the next stage!
+
+### ::{ flag=RU }:: [Skrowell](https://osu.ppy.sh/users/9694263) vs ::{ flag=RU }:: [Welter](https://osu.ppy.sh/users/11552867)
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/118799095)**\
+**[You can view the match VOD here!](https://youtu.be/fIpS-LFpB7w?si=qcziZXmLM-eatEO2&t=35337)**
+
+The final match of the tournament. ::{ flag=RU }:: [Welter](https://osu.ppy.sh/users/11552867), being the runner-up for the last 2 iterations of MOE, now gets a third chance to win the tournament. And even though ::{ flag=RU }:: [Skrowell](https://osu.ppy.sh/users/9694263) was not getting this far before and actually losing to ::{ flag=RU }:: [Welter](https://osu.ppy.sh/users/11552867) last year, he still has high chances to win this tournament as well.
+
+The first half of the match went pretty straightforward: both players were easily taking their picks up until the score of 3-3. After that, something went wrong and 3 breakpoints have happened in a row, getting the score to 4-5. The last map was picked by ::{ flag=RU }:: [Welter](https://osu.ppy.sh/users/11552867) and won by him as well!
+
+With a score of 6-4, ::{ flag=RU }:: [Welter](https://osu.ppy.sh/users/11552867) have won Moscow osu! Event for the first time, breaking his 2nd place curse. His performance during the whole tournament was very astonishing, making him one of the best Russian players with a no doubt!
+
+![Tournament winner (Welter)](/wiki/shared/news/2025-09-30-moscow-osu-event-2025-recap/welter_prize.jpg)
+
+# Other tournaments
+
+Besides the main gamemode, there were also the LAN Tournaments for other gamemodes, with osu!mania being separated to 4K and 7K. All matches were played live on the stage as well! All tournaments had a single-elimination bracket and began from the Semifinals.
```

Comment:
IMO this has better structure, with the most important bit finishing the paragraph

```suggestion
Game modes other than osu! also had their share of LAN tournaments, with osu!mania separated to 4K and 7K. All tournaments began from the Semifinals, had a single-elimination bracket, and were played live on the stage as well!
```